kytos/core/controller.py

```diff
@@ -251,14 +253,21 @@ def toggle_debug(self, name=None):
     def start(self, restart=False):
         """Create pidfile and call start_controller method."""
         self.enable_logs()
-        if self.options.database:
-            self.db_conn_or_core_shutdown()
-            self.start_auth()
-        if self.options.apm:
-            self.init_apm_or_core_shutdown()
-        if not restart:
-            self.create_pidfile()
-        self.start_controller()
+        # pylint: disable=broad-except
+        try:
+            if self.options.database:
+                self.db_conn_or_core_shutdown()
```

Comment:
There's room to also allow both `db_conn_or_core_shutdown` and `init_apm_or_core_shutdown` to reraise as mentioned on https://github.com/kytos-ng/kytos/issues/415, but I'll only address in the feature since it's just a chore, and that way it also minimizes the chunks of code that need to be backported.